src/lib/MetadataLib.sol

```diff
@@ -224,9 +563,20 @@ library MetadataLib {
         string memory closeQuote = quoted ? "\"" : "";
         string memory terminator = terminal ? "" : ",";
 
-        // Use single concat with all parts
         return string.concat(
             "{\"trait_type\": \"", trait, "\", \"value\": ", openQuote, value, closeQuote, "}", terminator
         );
     }
+
+    function toLockTag(Lock memory lock) internal pure returns (bytes12) {
```

Comment:
We should be consistent about prefixing internal functions (aside from in libraries) with an underscore